src/internal/utils.js

```diff
@@ -10,7 +10,7 @@ export const kFalse = konst(false)
 export const noop = () => {}
 export const ident = v => v
 
-export const isDev = process.env.NODE_ENV === 'development'
+export const isDev = typeof process !== "undefined" && process.env.NODE_ENV === 'development'
```

Comment:
actually it is not the only place we use `process`, there are few (like 2 or so) more

Do you want to update your PR or should I just do it? by mistake Ive already done it on my local branch, but wouldnt like to take this away from you, so let me know :)